server/graphql-engine.cabal

```diff
@@ -61,6 +61,7 @@ library
                      , wai
                      , postgresql-binary
                      , process
+                     , validation
```

Comment:
monad-validate might remove some boilerplate

Reply:
I tried doing this and TBH it didn't make much of a difference. So just leaving this as is atm.